jsapp/js/projects/projectsTable/projectsTableRow.module.scss

```diff
@@ -42,7 +42,7 @@ $projects-table-hover-bg: colors.$kobo-gray-98;
   &[data-field='ownerOrganization'],
   &[data-field='status'],
   &[data-field='submissions'] {
-    width: 12.5%;
+    width: 13%;
```

Comment:
This removed a spurious ellipsis for default screen width - which was really being caused I think, by invisible space being truncated. The real solve for this would probably involve being careful about whitespace at the component level, and may involve cooperation with [our formatter](https://prettier.io/docs/en/options.html#html-whitespace-sensitivity).